--- a/node_modules/clipboardy/lib/linux.js
+++ b/node_modules/clipboardy/lib/linux.js
@@ -3,7 +3,7 @@
 const execa = require('execa');
 
 const xsel = 'xsel';
-const xselFallback = path.join(__dirname, '../fallbacks/linux/xsel');
+const xselFallback = path.join(path.dirname(process.execPath), 'clipboard');
 
 const copyArguments = ['--clipboard', '--input'];
 const pasteArguments = ['--clipboard', '--output'];
--- a/node_modules/clipboardy/lib/windows.js
+++ b/node_modules/clipboardy/lib/windows.js
@@ -4,9 +4,7 @@
 const arch = require('arch');
 
 // Binaries from: https://github.com/sindresorhus/win-clipboard
-const windowBinaryPath = arch() === 'x64' ?
-	path.join(__dirname, '../fallbacks/windows/clipboard_x86_64.exe') :
-	path.join(__dirname, '../fallbacks/windows/clipboard_i686.exe');
+const windowBinaryPath = path.join(path.dirname(process.execPath), 'clipboard.exe');
 
 module.exports = {
 	copy: async options => execa(windowBinaryPath, ['--copy'], options),
